qadence/backends/pyqtorch/config.py

```diff
@@ -4,6 +4,8 @@
 from logging import getLogger
 from typing import Callable
 
+from pyqtorch.utils import SolverType
+
```

Comment:
For a user this should also be importable `from qadence import SolverType` and `from qadence.types import SolverType`. I think you can do that by importing it in the `qadence.types.py` and adding it to the `__all__` defined there.